.eslintrc

```diff
@@ -24,7 +24,8 @@
             }
         ],
         "jest/max-nested-describe": ["error", { "max": 4 }],
-        "jest/no-conditional-in-test": "warn",
+        "jest/no-conditional-in-test": "off",
+        "jest/no-conditional-expect": "off",
```

Comment:
turned these off as it is just too hard to follow with the current codebase and just makes it hard to read old tests.